CSharp/Library/Microsoft.Bot.Connector.Shared/StateAPI/Models/BotData.cs

```diff
@@ -21,9 +21,8 @@ public partial class BotData
         /// <summary>
         /// Initializes a new instance of the BotData class.
         /// </summary>
-        public BotData(string eTag = default(string), object data = default(object))
+        public BotData(string eTag = @"*", object data = default(object))
```

Comment:
>@"*" [](start = 37, length = 4)

default is "overwrite existing data because this node isn't aware of concurrent change made by other nodes" ?